src/libraries/Microsoft.PowerFx.Connectors/Public/ConnectorType.cs

```diff
@@ -148,9 +144,7 @@ internal ConnectorType(ISwaggerSchema schema, ISwaggerParameter openApiParameter
                 // SalesForce only
                 if (schema.ReferenceTo != null && schema.ReferenceTo.Count == 1)
                 {
-                    ExternalTables = new List<string>(schema.ReferenceTo);
-                    RelationshipName = schema.RelationshipName;
-                    ForeignKey = null; // SalesForce doesn't provide it, defaults to "Id"
+                    ExternalTables = new List<(string, string)>(schema.ReferenceTo.Select(r => (r, null as string /* Seems to always be Id */)));
```

Comment:
>null as string /* Seems to always be Id */ [](http://example.com/codeflow?start=99&length=42)

If its always Id, should this change from null -> "Id"?

Reply:
The problem is that nobody has been able to confirm if it's "Id" or the primary key (and if the primary key is always "Id" in SalesForce) or if it could be something else.
I have also searched the SalesForce dev docs w/o finding anything.

As a result I prefer keeping "null" to say "I don't know" rather than asserting, maybe wrongly", "it"s Id"